src/components/SlateEditor/RichTextEditor.jsx

```diff
@@ -14,13 +14,27 @@ import { Editor } from 'slate-react';
 import { isKeyHotkey } from 'is-hotkey';
 import BEMHelper from 'react-bem-helper';
 import { css } from '@emotion/core';
+import { hasNodeOfType } from './utils';
 import createSlateStore, { setSubmitted } from './createSlateStore';
 import { PluginShape } from '../../shapes';
 
 const isBoldHotkey = isKeyHotkey('mod+b');
-const isItalicHotkey = isKeyHotkey('mod+i');
-const isUnderlinedHotkey = isKeyHotkey('mod+u');
-const isSaveHotkey = isKeyHotkey('mod+s');
+const isCodeHotKey = isKeyHotkey('mod+k');
+const isCodeBlockHotKey = isKeyHotkey('mod+alt+k');
```

Comment:
Markert tekst blir ikke med til kode-modalen her.

Reply:
om man har trykket `mod+k` så funker det å skrive kode, men `mod+k` en gang til avslutter ikke kodeinnskrivingen

Reply:
Dette er felles oppførsel for alle marks. Det funker ikkje å oppheve en mark om du ikkje har valgt en tekst. Test det samme med bold og italics.

Reply:
Det er sant, da burde vi kanskje fikse det på ett eller annet tidspunkt?

Reply:
Fjerner codeblock fordi den ikkej er i toolbar.

Reply:
> Det er sant, da burde vi kanskje fikse det på ett eller annet tidspunkt?

Sant. Kan sjå om eg finn ut av det.

Reply:
Er faktisk en bug i slate: https://github.com/ianstormtaylor/slate/issues/2830

Reply:
Kan muligens komme rundt det med å nedgradere til slate 0.45 men er litt nølende til det.

Reply:
Er heller ikke så keen på å nedgradere slate. Da kan vi heller la det ligge til vi finner en annen løsning imo.